content/special-topics/known-issues.md

```diff
@@ -0,0 +1,46 @@
+---
+title: known issues
+id: known-issues
+weight: 90
+draft: false
+author: "people"
+---
+
+Although darktable aims at keeping compatibility with old edits, there are some limitations that you need to be aware of and cannot be solved in a programmatic way.
+
+# History inconsistencies with modules enabled by default
+
+## Context
+
+The editing history of pictures processed in darktable is saved in the database and in the sidecar XMP files stored in regard of your RAW files, in the same folder. This history records every step of settings change that happened during editing and let you go back to any step at any time.
+
+Before darktable 3.0, modules like *white balance*, *demosaic* and *base curve* that were enabled by default for technical reasons did not write an entry in the editing history if the user did not change the default settings. The software always assumed that no history meant default settings were used. This is not safe because it assumes defaults will never change in the future and it prevents a systematic handling of modules.
+
+darktable 3.0 introduced the pipeline reordering feature, allowing to change the order of application of modules in the pipe, which needs every module to write an entry in history for consistency and safety.
+
+When no entry is detected for a module mandatorily enabled by default, darktable 3.0 and later will initialize a new entry with the default parameters of the module as of the current darktable version.
+
+As a result, edits created with versions prior to 3.0 will lack some history entries which may be initialized by newer versions with different defaults than the ones they were created with, especially if you use the *scene-referred* workflow and the *modern* chromatic adaptation, and the look may change.
+
+## Solution
+
+When you open pictures edited with darktable 2.6 or before:
+
+1. check that the *color calibration* module is turned off. If it is on, it means the *white balance* module left no history entry, which means it was used with "as shot in camera" setting, fetching the white balance setting defined in the RAW metadata. In that case:
```

Comment:
I don't think that's true. My recollection is that color calibration will only ever be automatically switched on for new edits (those never opened in the darkroom before). The issue with older edits (where the white balance was left unchanged) is that color calibration is *off* and white balance is automatically initialised to D65. So the fix is _either_ to change the WB module to "as shot" _or_ to enable color calibration, depending on whether you want to preserve the old edit or not.

Reply:
I need to find that bug again, but I seem to recall it involved both.

Reply:
I've been experiencing this issue on a number of old edits fairly recently